app/src/editor/canvas/components/ModuleHelp.jsx

```diff
@@ -23,22 +28,73 @@ export default class ModuleHelp extends React.Component {
 
     async load() {
         const { moduleId } = this.props
+        const { moduleName } = this.props
+
         const help = await services.moduleHelp({
             id: moduleId,
         })
+
+        const cleanedName = moduleName.replace(/\s/g, '').replace(/\(/g, '_').replace(/\)/g, '')
```

Comment:
Why not just use the components here directly, e.g. just returning `<BooleanConstant />`? We could avoid doing the work twice.

Reply:
I could, but it would take a while!

The backstory to this is that the original module names had spaces and brackets inside them and the starting point was a JSON file containing all of them. So I needed to script-export 100s of these modules with a cleaned up filename to be JS-compliant. Here, when I dynamically import these files, I need to make a mirrored 'cleanup' of the original name + id to match those scripted file names.